File: micro-judge/src/io/topics.rs
const DEFAULT_MAKE_MOVE_CMD: &str = "bugout-make-move-cmd";
const DEFAULT_GAME_STATES_CHANGELOG: &str = "bugout-game-states";
const DEFAULT_MOVE_ACCEPTED_EV: &str = "bugout-move-accepted-ev";

#[derive(Clone)]
pub struct StreamTopics {
    pub make_move_cmd: String,
    pub game_states_changelog: String,
    pub move_accepted_ev: String,
}
impl Default for StreamTopics {
    fn default() -> Self {
        StreamTopics {
            make_move_cmd: DEFAULT_MAKE_MOVE_CMD.to_string(),
            game_states_changelog: DEFAULT_GAME_STATES_CHANGELOG.to_string(),
            move_accepted_ev: DEFAULT_MOVE_ACCEPTED_EV.to_string(),
        }
    }
}

File: judge/src/main/kotlin/Topics.kt
const val MAKE_MOVE_CMD_TOPIC = "bugout-make-move-cmd"
const val MAKE_MOVE_CMD_DEDUP_TOPIC = "bugout-make-move-cmd-dedup"
const val MOVE_ACCEPTED_EV_TOPIC = "bugout-move-accepted-ev"
const val MOVE_REJECTED_EV_TOPIC = "bugout-move-rejected-ev"
const val GAME_STATES_CHANGELOG_TOPIC = "bugout-game-states"
const val GAME_STATES_STORE = "bugout-game-states-store-judge"

const val MAKE_MOVE_DEDUP_STORE = "bugout-judge-make-move-cmd-dedup-store"

val ALL_TOPICS = arrayOf(GAME_STATES_CHANGELOG_TOPIC, MAKE_MOVE_CMD_TOPIC, MOVE_ACCEPTED_EV_TOPIC, MOVE_REJECTED_EV_TOPIC,MAKE_MOVE_CMD_DEDUP_TOPIC)
